Formula/sqlite.rb

```diff
@@ -50,6 +50,9 @@ def install
 
     system "./configure", *args
     system "make", "install"
+
+    # Avoid rebuilds of dependants that hardcode this path.
+    inreplace lib/"pkgconfig/sqlite3.pc", prefix, opt_prefix
```

Comment:
With this we should be able to get rid of `libgda` revbumps for future `sqlite` upgrades.